.github/workflows/template-build.yml

```diff
@@ -121,7 +121,6 @@ jobs:
           NEXT_PUBLIC_TINA_CLIENT_ID: ${{ secrets.NEXT_PUBLIC_TINA_CLIENT_ID }}
           NEXT_PUBLIC_TINA_BRANCH: ${{ github.head_ref || github.ref_name }}
           NEXT_PUBLIC_ZENDESK_CHAT_KEY: ${{ secrets.NEXT_PUBLIC_ZENDESK_CHAT_KEY }}
-          GOOGLE_RECAPTCHA_KEY: ${{ secrets.GOOGLE_RECAPTCHA_KEY }}
```

Comment:
everywhere else is a rename dropping `_v2` was this deletion intentional?

Reply:
Yes I deleted it because it is outdated and being replaced with the actual GOOGLE_RECAPTCHA_KEY from the App Settings. Moreover, it was referenced from GitHub secrets.